src/components/Lightbox/index.tsx

```diff
@@ -31,35 +33,53 @@ type LightboxProps = {
     /** Additional styles to add to the component */
     style?: StyleProp<ViewStyle>;
 
-    /** The index of the carousel item */
-    index?: number;
-
-    /** The index of the currently active carousel item */
-    activeIndex?: number;
-
-    /** Whether the Lightbox is used within a carousel component and there are other sibling elements */
-    hasSiblingCarouselItems?: boolean;
-
     /** Range of zoom that can be applied to the content by pinching or double tapping. */
     zoomRange?: Partial<ZoomRange>;
 };
 
 /**
  * On the native layer, we use a image library to handle zoom functionality
  */
-function Lightbox({
-    isAuthTokenRequired = false,
-    uri,
-    onScaleChanged,
-    onError,
-    style,
-    index = 0,
-    activeIndex = 0,
-    hasSiblingCarouselItems = false,
-    zoomRange = DEFAULT_ZOOM_RANGE,
-}: LightboxProps) {
+function Lightbox({isAuthTokenRequired = false, uri, onScaleChanged: onScaleChangedProp, onError, style, zoomRange = DEFAULT_ZOOM_RANGE}: LightboxProps) {
     const StyleUtils = useStyleUtils();
 
+    const isPagerScrollingFallback = useSharedValue(false);
```

Comment:
This one was hard to understand as well, so a comment would be nice

Reply:
Why would `AttachmentCarouselPagerContext` not be available?